app/models/manager_refresh/inventory/automation_manager.rb

```diff
@@ -0,0 +1,33 @@
+module ManagerRefresh::Inventory::AutomationManager
+  extend ActiveSupport::Concern
+  include Core
+
+  class Inventory < ManagerRefresh::Inventory::Core::Inventory
+    def credentials
```

Comment:
this will only work if `credentials` returns a lambda and looks up the approperiate `automate_manager / credentials` at runtime